appveyor.yml

```diff
@@ -9,6 +9,11 @@ init:
 
 clone_depth: 5
 skip_branch_with_pr: true
+skip_commits:
+  files:
+    - docs/
+    - '**/*.md'
+    - '**/*.html'
```

Comment:
`docs/` looks safe enough, but I am not too sure about the rest. In theory we could break tests by modifying Markdown or HTML files in the application itself. Which files where you thinking of?

Reply:
oof. You are right.  Let's limit to just the `docs/` folder then.